packages/ui/src/components/Text/index.tsx

```diff
@@ -19,6 +19,7 @@ const PROMINENCES = {
 
 type ProminenceProps = keyof typeof PROMINENCES
 type PlacementProps = React.CSSProperties['textAlign']
+type WhiteSpaceProps = React.CSSProperties['whiteSpace']
```

Comment:
I know it was there before but we avoid `React.` in our code base, instead you can import CSSProperties from react above